packages/hardhat/deploy/001-deploy-Spender.ts

```diff
@@ -0,0 +1,30 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { DeployFunction } from 'hardhat-deploy/types';
+import { devChains } from '../helper-hardhat-config';
+import { network } from 'hardhat';
+import verify from '../utils/verify/verify';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy, log } = deployments;
+
+  const { deployerSpender } = await getNamedAccounts();
+
+  const args: any[] = [];
+
+  const spender = await deploy('Spender', {
+    from: deployerSpender,
+    args,
+    log: true,
+    autoMine: true,
+  });
+
+  if (!devChains.includes(network.name)) {
+    log('------- verifying started --------');
```

Comment:
`log('Verifying...');`

Once we start with the `----- verifying started ----` the next is `!!!!! verifying failed !!!!` and we never win the competition over attention